frontend/src/app/projects/dashboard/metrics/page.tsx

```diff
@@ -0,0 +1,296 @@
+      handleAppError(graphQLRequestError)
+    }
+  }, [data, graphQLRequestError])
+
+  const orderingSections: DropDownSectionProps[] = [
+    {
+      title: 'Score',
+      items: [
+        { label: 'High to Low', key: 'scoreDESC' },
+        { label: 'Low to High', key: 'scoreASC' },
+      ],
+    },
+  ]
+  const filteringSections: DropDownSectionProps[] = [
+    {
+      title: 'Project Level',
+      items: [
+        { label: 'Incubator', key: 'incubator' },
+        { label: 'Lab', key: 'lab' },
+        { label: 'Production', key: 'production' },
+        { label: 'Flagship', key: 'flagship' },
+      ],
+    },
+    {
+      title: 'Project Health',
+      items: [
+        { label: 'Healthy Projects', key: 'healthy' },
+        { label: 'Projects Needing Attention', key: 'needsAttention' },
+        { label: 'Unhealthy Projects', key: 'unhealthy' },
+      ],
+    },
+    {
+      title: 'Reset Filters',
+      items: [{ label: 'Reset All Filters', key: 'reset' }],
+    },
+  ]
+
+  const getCurrentPage = () => {
+    return Math.floor(pagination.offset / PAGINATION_LIMIT) + 1
+  }
+
+  return (
+    <>
+      <div className="mb-4 flex items-center justify-between">
+        <h1 className="text-2xl font-bold">Project Health Metrics</h1>
+        <div className="flex items-center space-x-2">
+          <ProjectsDashboardDropDown
+            buttonDisplayName="Filter By"
+            icon={faFilter}
+            sections={filteringSections}
+            selectionMode="multiple"
+            selectedKeys={activeFilters}
+            onAction={(key: string) => {
+              // Because how apollo caches pagination, we need to reset the pagination.
+              const newPagination = { offset: 0, limit: PAGINATION_LIMIT }
+              setPagination(newPagination)
+              let newFilters = { ...currentFilters }
+              const newParams = new URLSearchParams(searchParams.toString())
+              if (key in healthFiltersMapping) {
+                newParams.set('health', key)
+                newFilters = { ...newFilters, ...healthFiltersMapping[key] }
+              } else if (key in levelFiltersMapping) {
+                newParams.set('level', key)
+                newFilters = { ...newFilters, ...levelFiltersMapping[key] }
+              } else {
+                newParams.delete('health')
+                newParams.delete('level')
+                newFilters = {}
+              }
+              setFilters(newFilters)
+              setActiveFilters(
+                Array.from(
+                  newParams
+                    .entries()
+                    .filter(([key]) => key != 'order')
+                    .map(([, value]) => value)
+                )
+              )
+              router.replace(`/projects/dashboard/metrics?${newParams.toString()}`)
+            }}
+          />
+
+          <ProjectsDashboardDropDown
+            buttonDisplayName="Sort By"
+            icon={faSort}
```

Comment:
Can we also update this to just say `score` for active order and change icon based on the state (asc/desc)? 
I was thinking we could use [this one for descending](https://fontawesome.com/icons/arrow-down-wide-short?f=classic&s=solid) and [this one for ascending](https://fontawesome.com/icons/arrow-up-wide-short?f=classic&s=solid)